Eradication/VectorPopulationIndividual.cpp

```diff
@@ -847,13 +846,18 @@ namespace Kernel
         {
             throw QueryInterfaceException( __FILE__, __LINE__, __FUNCTION__, "m_context->GetParent()", "IVectorSimulationContext", "ISimulationContext" );
         }
-        suids::suid current_node = m_context->GetSuid();
+        VectorGender::Enum     vector_gender = VectorGender::VECTOR_FEMALE;
+        Gender::Enum human_gender_equivalent = m_pMigrationInfoVector->ConvertVectorGender( vector_gender );
+        suids::suid             current_node = m_context->GetSuid();
         m_pMigrationInfoVector->UpdateRates( current_node, get_SpeciesID(), p_vsc );
-        m_pMigrationInfoVector->SetIndFemaleRates( 0 ); // setting for generic migration use
-        if( m_pMigrationInfoVector->GetMigrationAlleleCombinationsSize() == 1 )
+
+        // we don't need the actual fraction traveling, just the setting of totals etc functionality
+        m_pMigrationInfoVector->GetFractionTraveling( vector_gender, 0 ); 
+
+        if( !m_pMigrationInfoVector->TravelByAlleles() )
```

Comment:
Should be able to replace this with the CanTravel( Gender::Enum ) -- hmm.  could be just vector_gender.

Reply:
yup. Combined CanTravel and MightTravel into a MightTravel.